client/components/confirmDialog/confirmDialog.directive.js

```diff
@@ -0,0 +1,13 @@
+'use strict'
+
+angular.module('observatory3App')
+.controller('confirmDialogCtrl', function($scope,$uibModalInstance,msg){
```

Comment:
also please add a space after the comma. And it might be nice to have an optional param which is the title